### PR TITLE
MC-1084 - client filtering docs updated

### PR DESCRIPTION
**Pull request by promanenko**

### CODE REVIEW

File: docs/modules/monitor-imdg/pages/client-filtering.adoc [hunk 1]
```diff
@@ -16,53 +16,51 @@ members (**Enabled** status), or if the feature is disabled for the
...
-and the members start allowing any client to connect.
-* If the status to be deployed is **Enabled**, all entries of the
-matching lists from the Client Filtering Lists section are
-copied into the deployed client filtering list and applied to
+filtering type and to deploy any modifications made in client filtering
+lists to the deployed list available to all cluster members.
+
+* If the status is **Disabled** then **SAVE CONFIGURATION** button appears.
+Click on the **SAVE CONFIGURATION** to clear all client filtering lists
+from the cluster members and the members will start allowing any client to connect.
```
Comment (Serdaro) [inline docs/modules/monitor-imdg/pages/client-filtering.adoc:24]: Maybe better to simplify like below?

If the status is **Disabled**, then the **SAVE CONFIGURATION** button appears; click on it to clear all the client filtering lists from the cluster members. With this, the members will start allowing any client to connect.

File: docs/modules/monitor-imdg/pages/client-filtering.adoc [hunk 2]
```diff
@@ -16,53 +16,51 @@ members (**Enabled** status), or if the feature is disabled for the
...
+image:ROOT:ClientFilteringDeployConfiguration.png[Deploy Configuration Button]
+
 Once a cluster member receives the deployed client filtering list
 from the Management Center, it immediately applies the list to
 all currently connected clients and then uses it for newly connecting
 clients. Deny-listed clients may connect to another cluster
 if they are configured to support blue-green deployment. If they are not,
 they shut down when denied access to the cluster. Please see the
-https://docs.hazelcast.com/hazelcast/latest/clients/java.html#blue-green-deployment-and-disaster-recovery[Blue-Green Deployment and Disaster Recovery section]
+xref:imdg:clients:java.adoc#blue-green-deployment-and-disaster-recovery[Blue-Green Deployment and Disaster Recovery section]
```
Comment (Serdaro) [inline docs/modules/monitor-imdg/pages/client-filtering.adoc:44]: Let's change the link as xref:hazelcast:clients:java.adoc#blue-green-deployment-and-disaster-recovery

**Review by emre-aydin — APPROVED** [review]

File: docs/modules/monitor-imdg/pages/client-filtering.adoc [hunk 3]
```diff
@@ -108,43 +103,45 @@ Let's create a new filter list:
...
+list has a green status indicator next to the name, it means the list is currently deployed.
 
 image:ROOT:ClientFilteringExampleStep5.png[Example Step 5]
 
-Finally, let's deploy our changes by clicking the **Deploy Changes** button. 
-After a brief moment, we have our allow-list deployed.
-The **Deploy Changes** and **Summary** buttons become disabled again and our
-list has no cross in the **Deploy** column
-as there are no changes to be deployed.
+Now we can see currently deployed rules(from all lists) by hitting **REPORT** button:
```
Comment (Serdaro) [inline docs/modules/monitor-imdg/pages/client-filtering.adoc:143]: Now we can see the currently deployed rules (from all lists) by hitting the ...

File: docs/modules/monitor-imdg/pages/client-filtering.adoc [hunk 4]
```diff
@@ -16,53 +16,51 @@ members (**Enabled** status), or if the feature is disabled for the
...
+* If the status is **Disabled** then **SAVE CONFIGURATION** button appears.
+Click on the **SAVE CONFIGURATION** to clear all client filtering lists
+from the cluster members and the members will start allowing any client to connect.
+
+image:ROOT:ClientFilteringSaveConfiguration.png[Save Configuration Button]
+
+* If the status is **Enabled** then **DEPLOY CONFIGURATION**
+button appears. On **DEPLOY CONFIGURATION** all entries of the
+matching lists from the Filtering Lists section are applied to
 all cluster members. Matching lists are selected by
```
Comment (Serdaro) [inline docs/modules/monitor-imdg/pages/client-filtering.adoc:31]: Similar suggestion as the above:

If the status is **Enabled**, then the **DEPLOY CONFIGURATION** button appears; click on it to apply all the entries of matching lists from the Filtering Lists section to ...

**Review by Serdaro — APPROVED** [review]
Minor edit suggestions.

File: docs/modules/monitor-imdg/pages/client-filtering.adoc (hunk 4 above)
Comment (promanenko) [reply docs/modules/monitor-imdg/pages/client-filtering.adoc]: The labels match how they look on UI. I agree it looks inconsistent, but I've stayed with the case it actually appears on UI.

**Pull request merged**